.github/workflows/yarn.yaml

```
@@ -0,0 +1,30 @@
+name: yarn
+
+on:
+  - pull_request
```

Should we only do this once a PR has merged?

```suggestion
  - pull_request
      types: [closed]
```

---

Oh actually I see what this is doing. This will break Renovate as it will no longer rebase the change. Probably fine?

---

The PR build may fail without the update. Here's an example from last night where I had to append the commit updating the lock file before it could merge: https://github.com/cashapp/turbine/pull/214

---

Actually what I really want is to run this if any status in a Renovate PR build fails

---

I could make it manually run based on a command like the Spotless one

---

> I could make it manually run based on a command like the Spotless one

Yeah that sounds like a good compromise to me!

---

Probably need to start documenting these PR commands too 😉

---

Switched